cost-analyzer/templates/cost-analyzer-deployment-template.yaml

```diff
@@ -1139,10 +1076,8 @@ spec:
                 configMapKeyRef:
                   name: {{ template "cost-analyzer.fullname" . }}
                   key: kubecost-token
-            {{- if .Values.kubecostAggregator.enabled }}
             - name: WATERFOWL_ENABLED
```

Comment:
we should make a tech debt ticket to remove wherever this var is used in the code for the 2.0 launch

Reply:
I've tried to remove all references to it while maintaining backwards-compatibility using `aggregator.deployMethod` defined in helpers.tpl.